БЗ/Личности/неописанные личности/Simanouskiy.scs

```diff
@@ -0,0 +1,5 @@
+Simanouskiy
+=> nrel_main_idtf:
+	[Симановский] 
```

Comment:
Look. We're adding this identifiers to identify concrete person. Do you know how many people in the world with surname Simanouskiy?